From 8af8cac8669e04db24180a90b7e54bd081b9d365 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 23 Oct 2025 02:41:03 +0200
Subject: [PATCH] Specify NotShowIn instead on OnlyShowIn in Autostart

This aligns the autostart with the launcher, since it's a generic
GUI for xrandr, and not specific for the LXDE session.
---
 src/lxrandr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lxrandr.c b/src/lxrandr.c
index 8373e6b..6f667d7 100644
--- a/src/lxrandr.c
+++ b/src/lxrandr.c
@@ -551,7 +551,7 @@ static void save_configuration()
     g_key_file_set_string( kf, grp, "Name", _("LXRandR autostart") );
     g_key_file_set_string( kf, grp, "Comment", _("Start xrandr with settings done in LXRandR") );
     g_key_file_set_string( kf, grp, "Exec", cmd->str );
-    g_key_file_set_string( kf, grp, "OnlyShowIn", "LXDE" );
+    g_key_file_set_string( kf, grp, "NotShowIn", "GNOME;KDE;XFCE;MATE;" );
 
     data = g_key_file_to_data(kf, &len, NULL);
     file = g_build_filename(  g_get_user_config_dir(),
